dev-lang/luajit/luajit-2.1.1702296283.recipe

```diff
@@ -3,14 +3,15 @@ DESCRIPTION="Lua is a powerful, dynamic and light-weight programming \
 language. It may be embedded or used as a general-purpose, stand-alone \
 language. LuaJIT is a Just-In-Time Compiler (JIT) for it."
 HOMEPAGE="https://luajit.org/luajit.html"
-COPYRIGHT="2005-2015 Mike Pall"
+COPYRIGHT="2005-2023 Mike Pall"
 LICENSE="MIT"
-REVISION="2"
-SOURCE_URI="https://luajit.org/download/LuaJIT-2.1.0-beta3.tar.gz"
-CHECKSUM_SHA256="1ad2e34b111c802f9d0cdf019e986909123237a28c746b21295b63c9e785d9c3"
-SOURCE_DIR="LuaJIT-2.1.0-beta3"
+REVISION="1"
+SOURCE_URI="git+https://luajit.org/git/luajit.git#29b0b282f59ac533313199f4f7be79490b7eee51"
```

Comment:
I saw there is a release also at their github page, maybe that could be used instead (or a srcGitRev tag)?

Reply:
Reasoning? unsafe sources are disabled by default so buildmaster wont pick this up for a build:
`Error: Downloading from unsafe sources is disabled in haikuports.conf!`

Reply:
This works (commit is the same):
```
srcGitRev="29b0b282f59ac533313199f4f7be79490b7eee51"
SOURCE_URI="https://github.com/LuaJIT/LuaJIT/archive/$srcGitRev.tar.gz"
CHECKSUM_SHA256="e322ab3d39d722fa06076258a495dee8cd11b9fdfbff7608d19b60b6e378e206"
```